### PR TITLE
Protfolio mobile version

### PR DESCRIPTION
**Pull request by mrkamin**
Portfolio Mobile Version final PR

### CODE REVIEW

**Comment by mrkamin** [comment]
PortFolio mobile version PR

**Review by cwaku — COMMENTED** [review]
Hi @mrkamin 

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

It seems like you have submitted the wrong PR URL. It shows to me in the dashboard that this is a re-review but otherwise here and there is no note that a previous reviewer requested that you make a new PR. Kindly resubmit.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

@cwaku 
thanks for your good and valuable feedback. sorry it is my mistake.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,651 @@
...
+  background: #6070ff;
+  border-top-left-radius: 25%;
+  min-width: 375px;
+  background-position: 265px;
+}
+
+.mobile-background {
+  background-image: url("/image/contact\ form\ shapes-mobile.svg");
+  background-repeat: no-repeat;
+  width: 600px;
```
Comment (jaflih) [inline style.css:545]: Please correct your responsive in mobile view.
I can scroll right/left but it is not in the requirement.
Capture d’écran 2022-09-30 à 18 04 02

File: style.css [hunk 2]
```diff
@@ -0,0 +1,651 @@
...
+  border-radius: 8px;
+  color: #fff;
+}
+
+.get-touch:hover {
+  background-color: #6070ff;
+  box-shadow: 0 8px 16 rgb(64, 83, 252, 0.24);
+  border-radius: 8px;
+  color: #fff;
+}
```
Comment (jaflih) [inline style.css]: Please refer to the template guide to have a design close to Figma.


When the user hovers or presses the button, we. want this effect 
image

**Review by jaflih — CHANGES_REQUESTED** [review]
Hi @mrkamin  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✔️ Linters passed 💯
✔️ The content is aligned with flexbox 🥇 
✔️ Great commit history

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @jaflih in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mrkamin** [comment]
correction RWD

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hello @mrkamin  👋,

_This is an additional review as a result of the second opinion request._

As mentioned by the previous reviewer, please follow the figma design for buttons events:

Screen Shot 2022-10-01 at 19 04 24

Currently what you have is 
Screen Shot 2022-10-01 at 19 04 24


Keep rocking 🚀! // can be customized

@rukundoeric 
thanks for your good and valuable feedback.
I will try my best to fix the issue.

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @mrkamin          :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the requested changes good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

- [ ] [OPTIONAL] To fix the horizontal scroll, you can consider adding the below lines of code in your style.css file.

```
html,
body {
  overflow-x: hidden;
}
```

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

@Nemwel-Boniface 
thanks for your good and valuable feedback and suggestions. 
I will consider your suggestions in my upcoming projects.

File: style.css (hunk 1 above)
Comment (mrkamin) [reply style.css:545]: @jaflih 
thanks for your good and valuable feedback.

File: style.css (hunk 2 above)
Comment (mrkamin) [reply style.css]: @jaflih 
thanks for your good and valuable feedback. 
I will consider your feedback in my next attempt.

**Pull request merged**